stl/src/stdhndlr.cpp

```diff
@@ -3,11 +3,13 @@
 
 // set_new_handler
 
+#include <mutex>
 #include <new.h>
 #include <new>
 
 namespace {
     _STD new_handler _New_handler;
+    constinit std::mutex _New_handler_mutex;

```

Comment:
Style nitpick: `stl/src` is somewhat inconsistent, but `_STD` appears immediately above, so let's use that here.